spec/state_machines/tax_return_state_machine_spec.rb

```diff
@@ -21,7 +21,7 @@
 
```

Comment:
sorry if i just lost track of this and it does exist somewhere but: is there a test that checks that the transition stays in sync with `tax_return.status` and `tax_return.state`? for some reason was expecting to see it here but fine if it lives somewhere else